tensorflow_datasets/core/registered.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, name, is_abstract=False, in_development=False):
 class RegisteredDataset(abc.ABCMeta):
   """Subclasses will be registered and given a `name` property."""
 
-  def __new__(cls, cls_name, bases, class_dict):
+  def __new__(cls, cls_name, bases, class_dict): # pylint: disable= C0204, W0221
```

Comment:
Please use the human readable names, rather than symbols `C0204` (e.g. `import-not-on-top`, `variable-redefined`)